src/cli/util.js

```diff
@@ -398,6 +398,10 @@ function createIgnorerFromContextOrDie(context) {
 }
 
 function eachFilename(context, patterns, callback) {
+  // workaround for fast-glob on Windows ref:
+  // https://github.com/mrmlnc/fast-glob#how-to-write-patterns-on-windows
+  patterns = patterns.map(path => path.replace(/\\/g, "/"));
```

Comment:
Please wrap this into `if (path.sep === "\\")`.

Reply:
Done.